plugins/sigma.plugins.design/sigma.plugins.design.js

```diff
@@ -1001,6 +1001,78 @@
     };
 
     /**
+     * Deletes styles from an element according to the specified options.
+     * If the only the nodeId is specified, this will delete the 'size', 'color', 'type', 'icon' and
+     * 'image'
+     *
+     * @param {String} target - The data target. Available values: "nodes", "edges".
+     * @param {Number|String} id - id of the Element to update
+     * @param {Object} [key] - property key to delete styles from.
+     */
+    this.deleteElementStyles = function(target, id, key){
+
+      if (id === undefined || id === null){
+        throw new Error('"design.deleteNodeStyles": nodeId is undefined');
+      }
+      if (target !== 'nodes' && target !== 'edges') {
+        throw new Error('"design.deleteNodeStyles": Unknown target ' + target);
+      }
+
+      if (!key){
+        var element;
+        if (target === 'nodes') {
+          element = s.graph.nodes(id);
+        } else {
+          element = s.graph.edges(id);
+        }
+
+        delete element.size;
+        delete element.type;
+        delete element.color;
+        delete element.icon;
+        delete element.image;
```

Comment:
My use case is indeed to remove a single node edge/property. 

However, one could want to remove all styles from an element for reason x or y. This could be used for this.